build/tc479540/test-buildconfig-docker.json

```diff
@@ -41,7 +41,7 @@
       "dockerStrategy": {
         "from": {
           "kind": "DockerImage",
-          "name": "docker.io/qeopenshift/ruby-22-centos7"
+          "name": "docker.io/qeopenshift/ruby-25-centos7"
```

Comment:
Please change to use openshift imagestream like above, if you use this image directly, the case will be failed in disconnected cluster; but if use openshift imagestream, we can mirror/import ruby image.